http4k-contract/src/main/kotlin/org/http4k/contract/openapi/v3/jacksonExt.kt

```diff
@@ -66,12 +66,12 @@ class JacksonJsonNamingAnnotated(private val json: ConfigurableJackson = Jackson
 
 object JacksonFieldMetadataRetrievalStrategy : FieldMetadataRetrievalStrategy {
     override fun invoke(target: Any, fieldName: String): FieldMetadata =
-        FieldMetadata(description = target.javaClass.findPropertyDescription(fieldName))
+        FieldMetadata(description = findPropertyDescription(target.javaClass, fieldName))
 
-    private fun Class<Any>.findPropertyDescription(name: String): String? =
-        kotlin.constructors.first().parameters
+    fun findPropertyDescription(clazz: Class<Any>, name: String): String? =
```

Comment:
AFAIK there's no need to change this from extension to 'regular' function. Just marking this as public (un-marking as private) should be enough to make this callable from other places.

Personally I too prefer the regular format over extension, but here I'd advocate for smaller diff.

Reply:
Unfortunately it is, otherwise it is not usable from the `JavaXValidationFieldRetrievalStrategy`. See it is imported and used there.

Reply:
I do see that. I wasn't sure before writing this comment so I pulled your code in a tried myself. You can happily import and use extension functions from other packages as long as they're public.

Reply:
Interesting, for some reason it didn't work for me when implementing it. Will fix.